romeo/asset-contract/package.json

```
@@ -0,0 +1,23 @@
+{
+  "name": "asset-contract",
+  "version": "1.0.0",
+  "main": "index.js",
+  "author": "friedger",
+  "license": "MIT",
+  "type": "module",
+  "scripts": {
+    "generate:docs": "npx tsx ./scripts/generate-docs.ts",
+    "test": "vitest run",
+    "test:coverage": "vitest run -- --coverage true",
+    "test:watch": "chokidar \"unit-tests/**/*.ts\" \"contracts/**/*.clar\" -c \"npm t\""
+  },
+  "dependencies": {
+    "@hirosystems/clarinet-sdk": "^0.6.4",
```

```suggestion
    "@hirosystems/clarinet-sdk": "^0.9.0",
```

(will require to re-run `npm install` to apply changes in the package-lock.json)